{
  "totalNumberSkippingTests": 0,
  "bugType": "only_fail",
  "totalNumberErroringTests": 0,
  "repo": "dotwebstack/dotwebstack-framework",
  "bugCommit": "https://github.com/dotwebstack/dotwebstack-framework/commit/8e4eacaab9b9",
  "failingModule": "/root/workspace/dotwebstack/dotwebstack-framework/347266570/core",
  "hostname": "spirals-vortex.lille.inria.fr",
  "totalNumberFailingTests": 1,
  "error-types": [
    "java.lang.AssertionError"
  ],
  "totalNumberRunningTests": 123,
  "failing-test-cases": [
    {
      "className": "org.dotwebstack.framework.informationproduct.template.TemplateProcessorTest",
      "failingMethods": [
        "org.dotwebstack.framework.informationproduct.template.TemplateProcessorTest#processString_ThrowsException_ForMissingParameter"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "\nExpected: an instance of org.dotwebstack.framework.informationproduct.template.TemplateException\n     but: \u003cjava.lang.NullPointerException: parameters\u003e is a java.lang.NullPointerException\nStacktrace was: java.lang.NullPointerException: parameters\n at org.dotwebstack.framework.informationproduct.template.TemplateProcessor.processString(TemplateProcessor.java:39)\n at org.dotwebstack.framework.informationproduct.template.TemplateProcessorTest.processString_ThrowsException_ForMissingParameter(TemplateProcessorTest.java:31)\n at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n at java.lang.reflect.Method.invoke(Method.java:498)\n at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\n at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\n at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\n at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\n at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)\n at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:239)\n at org.junit.rules.RunRules.evaluate(RunRules.java:20)\n at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)\n at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)\n at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)\n at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)\n at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)\n at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)\n at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)\n at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)\n at org.junit.runners.ParentRunner.run(ParentRunner.java:363)\n at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:369)\n at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:275)\n at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:239)\n at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:160)\n at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)\n at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)\n at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)\n at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)\n",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    }
  ]
}